src/main.py

```diff
@@ -0,0 +1,42 @@
+from leader import Leader
+from follower import Follower
+from tagrec import TagRecognition
+from inputcontroller import InputController
+
+
+"""
+Main script to run follower and leader functionality
```

Comment:
Try to be more descriptive and concise in the first line. Also end in a period always.
Maybe something like "Script to decide leader and follower roles." Then explain more.